### PR TITLE
Fix #234

### LINKED ISSUE #234: Support `drop schema if exists xxx`
```
mysql -u root -h 127.0.0.1 -P 4000 test
mysql> drop schema if exists xxx;
ERROR 1105 (HY000): line 1 column 6 near "schema"
```

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
Support `drop schema if exists xxx`
#234

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -1221,11 +1223,18 @@ DeleteFromStmt:
 		}
 	}
 	
+DropDatabase:
```
Comment (siddontang) [inline parser/parser.y]: I think only choose "DATABASE" and "SCHEMA", no "DROP" here may be better.

@shenli

Comment (shenli) [inline parser/parser.y]: Define a DatabaseSym var:
DatabaseSym:
"DATABASE" | "SCHEMA"

Also check CreateDatabase please.

Comment (siddontang) [inline parser/parser.y]: +1

Comment (qiuyesuifeng) [inline parser/parser.y]: Yes, that is better.

**Comment by siddontang** [comment]
LGTM

File: tidb_test.go [hunk 2]
```diff
@@ -884,6 +884,19 @@ func (s *testSessionSuite) TestBootstrap(c *C) {
...
+	store := newStore(c, s.dbName)
+	se := newSession(c, store, s.dbName)
+
+	mustExecSQL(c, se, "create database xxx")
+	mustExecSQL(c, se, "use xxx")
+	mustExecSQL(c, se, "drop database xxx")
+
+	mustExecSQL(c, se, "create schema xxx")
+	mustExecSQL(c, se, "use xxx")
+	mustExecSQL(c, se, "drop schema xxx")
```
Comment (shenli) [inline tidb_test.go]: Any test case for droping an unexists schema with/without IfExists clasue?

**Pull request merged**